sycl/doc/design/CompileTimeProperties.md

```diff
@@ -465,25 +465,48 @@ When the device compiler generates code to reference the decorated member
 variable, it emits a call to the LLVM intrinsic function
 [`@llvm.ptr.annotation`][10] that annotates the pointer to that member
```

Comment:
I happened to notice that this footnote [10] is no longer defined because you deleted it below.

Reply:
My bad! It should be back now.

Reply:
Thanks.